rflx/pyrflx/typevalue.py

```diff
@@ -582,18 +623,26 @@ def _get_length(self, fld: str) -> Optional[Number]:
             if (
                 self._fields[l.source.name].set
                 and l.length != UNDEFINED
-                and self.__simplified(l.condition) == TRUE
+                and (self._skip_verification or self.__simplified(l.condition) == TRUE)
             ):
                 length = self.__simplified(l.length)
                 return length if isinstance(length, Number) else None
         return None
 
     def _get_first(self, fld: str) -> Optional[Number]:
         for l in self._type.incoming(Field(fld)):
-            if l.first != UNDEFINED and self.__simplified(l.condition) == TRUE:
+            if l.first != UNDEFINED and (
+                self._skip_verification or self.__simplified(l.condition) == TRUE
+            ):
                 first = self.__simplified(l.first)
                 return first if isinstance(first, Number) else None
         prv = self._prev_field(fld)
+        if self._skip_verification and prv:
+            first = self._fields[prv].first
+            length = self._fields[prv].typeval.size
+            assert isinstance(first, Number)
+            assert isinstance(length, Number)
+            return Number(first.value + length.value)
```

Comment:
That should be the same as `return first + length`.

Reply:
Fixed in https://github.com/Componolit/RecordFlux/pull/405/commits/30369da5614566660776abbb5d84cc28ea8bd21e.